whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -206,6 +206,12 @@ public RdapObject mapError(final int errorCode, final String errorTitle, final L
         return rdapObject;
     }
 
+    public RdapObject mapAutnumError(final int errorCode, final String errorTitle, final List<String> errorDescriptions){
+        final RdapObject rdapObject = mapError(errorCode, errorTitle, errorDescriptions);
+        rdapObject.getRdapConformance().add(RdapConformance.FLAT_MODEL.getValue());
+
```

Comment:
do we need to add. FLAT_MODEL conformance only incase of error?

Reply:
I mean when we create an Autnum response we are already adding RdapConformance.FLAT_MODEL.getValue(). If we add it before performing the query any exception will have this value in that case we do not need to create a autnumexception?

Reply:
I have checked we do not have that information available in mapper class , it creates an empty bject incase of error